app/src/main/java/com/futurice/freesound/functional/Functions.java

```diff
@@ -41,6 +58,25 @@
         return NOTHING1;
     }
 
+    /**
+     * Wraps the call to the function in try-catch and propagates thrown
+     * checked exceptions as RuntimeException.
+     * @param <T> the first input type
+     * @param <U> the second input type
+     * @param <R> the output type
+     * @param f the function to call, not null (not verified)
+     * @param t the first parameter value to the function
+     * @param u the second parameter value to the function
+     * @return the result of the function call
+     */
+    public static <T, U, R> R apply(BiFunction<T, U, R> f, T t, U u) {
+        try {
+            return f.apply(t, u);
+        } catch (Exception ex) {
+            throw ExceptionHelper.wrapOrThrow(ex);
```

Comment:
`ExceptionHelper.wrapOrThrow(ex)` is an internal RxJava API, so we should create our own version.